src/freenet/support/SectoredRandomGrabArray.java

```diff
@@ -182,7 +183,8 @@ private RandomGrabArrayItem removeRandomLimited(
 			if(excludeTime > 0) {
 				excluded++;
 				if(excluded > MAX_EXCLUDED) {
-					Logger.normal(this, "Too many sub-arrays are entirely excluded on "+this+" length = "+grabArrays.length, new Exception("error"));
+					if (logMINOR)
+                        Logger.minor(this, "Too many sub-arrays are entirely excluded on "+this+" length = "+grabArrays.length, new Exception("error"));
```

Comment:
You’re using whitespace that does not match the surrounding area here.

Reply:
yikes, thank you. That’s the tab vs. 4 spaces …

Reply:
Yeah, as usual… IDEA can auto-detect and -adapt to existing whitespace conditions! 😄